Automattic/jetpack
# Fix/20806 Browser console deprecation warning for BlcokControl entity

Fixes #20806

#### Changes proposed in this Pull Request:

* There was a browser console warning that BlockControl is deprecated as a part of `@wordpress/editor` module
* I have modified the import statement to `@wordpress/block-editor` so that warning won't appear again.
* Looked for other places but didn't found any such or similar occurrences.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use? NO



#### Testing instructions:






* Checkout to the PR branch
* Create a new post
* Open browser console
* Add Twitter block and embed any tweet link 
* Observe in the browser console there are no such deprecation errors.

I'm attaching both the screenshots Before and After changes
Before Changes
![before changes](https://user-images.githubusercontent.com/21127788/131736283-267e7064-c28d-4efd-a92a-836c98a21360.png)

After Changes
![after changes](https://user-images.githubusercontent.com/21127788/131736331-a402e416-1a32-465c-96d5-dd0c4e4d798d.png)

cc @jeherve
